handlers/main.yml

```diff
@@ -3,4 +3,8 @@
   service:
     name: "{{slurm_service}}"
     state: reloaded
-  when: slurm_service_enabled | bool
```

Comment:
Don't you need this condition, and also on the next handler, to ensure you don't start slurm or munge when disabled?

Reply:
True - I thought Ansible was complaining about conditionals in handlers, but actually it doesn't seem to mind.